packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -22,17 +21,17 @@ function BlockListAppender( {
 	rootClientId,
 	canInsertDefaultBlock,
 	isLocked,
-	renderAppender,
+	renderAppender: CustomAppender,
 } ) {
 	if ( isLocked ) {
 		return null;
 	}
 
 	// A render prop has been provided, use it to render the appender.
-	if ( renderAppender ) {
+	if ( CustomAppender ) {
 		return (
 			<div className="block-list-appender">
-				{ createElement( renderAppender ) }
+				<CustomAppender />
```

Comment:
@aduth - this is another alternative since JSX desugars to `createElement`.

Reply:
> @aduth - this is another alternative since JSX desugars to `createElement`.

I hadn't thought about that, but yes, I think that'd make for a fine (preferred) alternative.